internal/cmd/operator-sdk/cleanup/cmd.go

```diff
@@ -37,15 +38,17 @@ func NewCmd() *cobra.Command {
 			return cfg.Load()
 		},
 		Run: func(cmd *cobra.Command, args []string) {
-			u := operator.NewUninstall(cfg)
 			u.Package = args[0]
-			u.DeleteAll = true
 			u.DeleteOperatorGroupNames = []string{operator.SDKOperatorGroupName}
 			u.Logf = log.Infof
 
 			ctx, cancel := context.WithTimeout(cmd.Context(), timeout)
 			defer cancel()
 
+			if err := u.Validate(); err != nil {
+				log.Fatalf("Flag validation error: %v\n", err)
+			}
```

Comment:
Validation shouldn't affect timeout
```suggestion
			if err := u.Validate(); err != nil {
				return fmt.Errorf("invalid command options: %v", err)
			}
			
			ctx, cancel := context.WithTimeout(cmd.Context(), timeout)
			defer cancel()
```